api/page-index/page-index.js

```diff
@@ -274,6 +282,10 @@ module.exports = [
               {
                 title: "interpolate",
                 href: "interpolate"
+              },
```

Comment:
Should we put `time_bucket_ng` right under `time_bucket` above?  (That feels right to me?)

If you are instead trying to put all experimental features at the end of the section -> then just one issue is I think this is incorrectly placed as a child of "gapfilling" in the hierarchy -> needs to be moved up one level.

Reply:
Ah, good spot!